creds/src/lib.rs

```diff
@@ -329,10 +436,19 @@ pub fn create_show_proof(client_state: &mut ClientState<ECPairing>, range_pk : &
     let show_groth16 = client_state.show_groth16(Some(context_str.as_bytes()), &io_types);
     
     // Create fresh range proof 
-    let time_sec = SystemTime::now()
```

Comment:
Suggest making a function in utils.rs to get the system time, we need timestamps in a couple places, it should be simpler.

Reply:
Created a new utils function to return seconds from utc now()